settings.py

```diff
@@ -1672,6 +1672,9 @@ def _determine_couch_databases(couch_databases):
     'ICDS_SUPERVISOR_FROM_AWC_OWNER':
         ['custom.icds.messaging.custom_recipients.supervisor_from_awc_owner',
          "ICDS: Supervisor Location from AWC Owner"],
+    'ICDS_PARENT_OF_CCS_RECORD':
+        ['custom.icds.messaging.custom_recipients.parent_of_ccs_record',
+         "ICDS: Parent of CCS record"],
```

Comment:
similarly could we rename this to "ICDS: Mother person case from ccs_record case"